h/static/scripts/filter/document-title.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+module.exports = [function() {
```

Comment:
Ditto. The array injection whatnot a) only matters if you want to inject something, and b) isn't necessary anymore because you can just mark a function with `// @ngInject`.